packages/parser-javascript/tests/fixtures/example.jsx

```diff
@@ -0,0 +1,14 @@
+let circle = document.createElement('circle');
```

Comment:
Note to self: Remove this file. I was using it to manually verify the end-to-end experience in the VS Code extension. Separate tests capturing individual conditions here are now part of `parser-jsx/tests`.